packages/customize-widgets/src/controls/inspector-section.js

```diff
@@ -57,5 +59,16 @@ export default function getInspectorSection() {
 				allowMultiple: true,
 			} );
 		}
+		collapse( opt ) {
```

Comment:
Let's rename `opt` to `option` or something more descriptive. We try our best to avoid abbreviations in Gutenberg.